wren-launcher/commands/launch.go

```diff
@@ -146,21 +186,29 @@ func Launch() {
 	pterm.Info.Println("Preparing project directory")
 	projectDir := prepareProjectDir()
```

Comment:
You can move this line to the beginning of this method, then L189-196 can be moved to L158